packages/drivers/odsp-driver/src/compactSnapshotParser.ts

```diff
@@ -116,50 +116,92 @@ function readTreeSection(node: NodeCore) {
 		 */
 		const length = treeNode.length;
 		if (length > 0 && treeNode.getMaybeString(0) === "name") {
-			if (length === 4) {
-				const content = treeNode.getMaybeString(2);
-				// "name": <node name>
-				// "children": <blob id>
-				if (content === "children") {
-					const result = readTreeSection(treeNode.getNode(3));
-					trees[treeNode.getString(1)] = result.snapshotTree;
-					slowTreeStructureCount += result.slowTreeStructureCount;
-					continue;
+			switch (length) {
+				case 4: {
+					const content = treeNode.getMaybeString(2);
+					// "name": <node name>
+					// "children": <blob id>
+					if (content === "children") {
+						const result = readTreeSection(treeNode.getNode(3));
+						trees[treeNode.getString(1)] = result.snapshotTree;
+						slowTreeStructureCount += result.slowTreeStructureCount;
+						continue;
+					}
+					// "name": <node name>
+					// "value": <blob id>
+					if (content === "value") {
+						snapshotTree.blobs[treeNode.getString(1)] = treeNode.getString(3);
+						continue;
+					}
+					break;
 				}
-				// "name": <node name>
-				// "value": <blob id>
-				if (content === "value") {
-					snapshotTree.blobs[treeNode.getString(1)] = treeNode.getString(3);
-					continue;
-				}
-			}
+				case 6: {
+					// "name": <node name>
+					// "nodeType": 3
```

Comment:
nodeType": 3 was a bug and I think SPO wanted to fix it.
I hope it's gone. If so, let's remove it.

Reply:
William mentioned it was not a bug. nodeType represents that it is a blob.